sdk/eventhub/event-hubs/samples/javascript/iothubConnectionString.js

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/*
+  This sample demonstrates how to convert an Iot Hubs connection string to
+  an Event Hubs-compatible connection string that can be used with the EventHubConsumerClient
+  to process events from an Iot Hub.
+*/
+
+const crypto = require("crypto");
```

Comment:
Quick check - is this just an auto-gen from the .ts file? Not going to both reviewing it if it is.

Reply:
I actually wrote the JavaScript one first then wrote the TypeScript one. It's almost exactly the same except the TypeScript one has a type guard function added for AmqpError.